src/app/components/settings/settings.component.ts

```diff
@@ -46,12 +46,14 @@ export class SettingsComponent implements OnInit {
     private electronService: ElectronService,
     private modalService: ModalService,
     private translate: TranslateService
-  ) {}
+  ) { }
 
   ngOnInit(): void {
     this.additionalInput = this.appState.addtionalExtensions;
   }
-
+  ngOnChanges(): void {
+    document.getElementById('settings').scrollTop = 0;
```

Comment:
I added `console.log()` above this and can confirm that this method gets called every time you switch between the _Tabs_ in _Settings_, but it doesn't seem to actually scroll to the top 😕 

ps, when you add `ngOnChanges()` method to an Angular class, you should also add it to the class declaration:
```ts
export class SettingsComponent implements OnInit, OnChanges {
```